public/server/db.json

```diff
@@ -70,6 +70,45 @@
     "lol":{
         "definition": "Laugh Out Loud"
     },
+
+    "stfu":{
+        "definition": "Shut The Fuck Up"
+    },
+
+    "ngl":{
+        "definition": "Not Gonna Lie"
+    },
+
+    "hru":{
+        "definition": "How Are You"
+    },
+
+    "srly":{
+        "definition": "Seriously"
+    },
+
+    "bff":{
+        "definition": "Best Friend Forever"
+    },
+
+    "tl":{
+        "definition": "Time Line"
+    },
+
+    "cl":{
+        "definition": "Contact List"
+    },
+
+    "lmao":{
```

Comment:
`lmao` is already being worked upon by someone. Please remove it so that when I merge their pull request, it won't appear twice in this file.

Reply:
I’ve made the necessary modification, kindly give it a look.